docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,312 @@
+            must be set as well.
+            3. `value` (OPTIONAL, string). The value of the feature. If `name` is set, this field must be set as wel.
+            For example: "male".
+
+    4. `measurements`.
+
+        1. `bar_plots` (list). The purpose of this field is to capute bar plot like measurements, for example SHAP values.
+        There can be multiple bar plots. For each bar plot the following fields are present.
+
+            1. `type` (REQUIRED, string). The type of bar plot, for example "SHAP".
+            2. `name` (OPTIONAL, string). A pretty name for the plot, for example "Mean Absolute SHAP Values".
+            3. `results` (list). The contents of the bar plot. A result represents a bar. There can be mutiple results.
+            For each result the following fields are present.
+                1. `name` (REQUIRED, string). The name of bar.
+                2. `value` (REQUIRED, float). The value of the corresponding bar.
+
+        2. `graph_plots` (list). The purpose of this field is to capture graph plot like measurements, such as partial dependence
+        plots.
+        There can be multiple graph plots. For each graph plot the following fields are present.
+
+            1. `type` (REQUIRED, string). The type of the graph plot, for example "partial_dependence".
+            2. `name` (OPTONAL, string). A pretty name of the graph, for example "Partial Dependence Plot".
+            3. `results` (list). Results contains the graph plot data. Each graph can depend on a specific output
+            class and feature. There can be multiple results. For each result the following fields are present.
+                1. `class` (OPTIONAL, string/int/float/bool). The output class name that the graph corresponds to.
+                This field is not always present.
+                2. `feature` (REQUIRED, string). The feature the graph corresponds to. This is required, since all
+                relevant graphs are dependend on features.
+                3. `data` (list)
+                    1. `x_value` (REQUIRED, float). The $x$-value of the graph.
+                    2. `y_value` (REQUIRED, float). The $y$-value of the graph.
+
+## Example
+
+### System Card
+
+```yaml
+upl: {upl_uri}                                          # Optional. Example: https://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland
+owners:
+- organization: {organization_name}                     # Required. Example: BZK
+  oin: {oin}                                            # Optional. Example: 00000001003214345000
+  name: {owner_name}                                    # Optional. Example: John Doe
+  email: {owner_email}                                  # Optional. Example: johndoe@email.com
+  role: {owner_role}                                    # Optional. Example: Data Scientist.
+
+# Model Cards
+models:
+ - !include {model_card_uri}                            # Optional. Example: model_card.yaml.
```

Comment:
Curious: is this an supported way of including yaml? Or should we consider using `include:` with a reference to a model card file or model_card_uri?

Reply:
There is no standard way of include's or import's in yaml that is out-of-the-box supported. There exists libraries to handle things like !include.